wazo_chatd/plugins/presences/bus_consume.py

```diff
@@ -5,11 +5,17 @@
 
 from wazo_chatd.exceptions import UnknownUserException
 from wazo_chatd.database.helpers import session_scope
-from wazo_chatd.database.models import Endpoint, Line, Session, Tenant, User
+from wazo_chatd.database.models import Line, Session, Tenant, User
 from .initiator import DEVICE_STATE_MAP, extract_endpoint_name
 
 logger = logging.getLogger(__name__)
 
+INUSE_STATE = (
+    'holding',
+    'ringing',
+    'talking',
+)
+
```

Comment:
Would it be better to import INUSE_STATE from initiator.py?

Reply:
Fixed
Thanks